nerdlandbot/translations/Translations.csv

```diff
@@ -1,131 +1,136 @@
+"remove_purger_usage,""<Channel ID>"",""<Kanaal ID>""";;;;;;
+"remove_purger_help,""**admin-only** \n \u2003 Stop the automatic removal of messages in the channel."",""**admin-functie** \n \u2003 Stop het automatisch verwijderen van berichten in het kanaal.""";;;;;;
+"list_purger_help,""List all channels with automatic message removal."",""Lijst alle kanalen op waar berichten automatisch worden verwijderd.""";;;;;;
+"purger_permissions,""It seems I don't have permission to remove messages in this channel."",""Het ziet er naar uit dat ik geen toestemming heb om berichten te verwijderen in dit kanaal.""";;;;;;
+"purger_added,""Automatically removing messages older than {1} day(s) in <#{0}>."",""Vanaf nu worden berichten ouder dan {1} dag(en) automatisch verwijderd in <#{0}>.""";;;;;;
+"purger_exists,""Messages already automatically get removed in <#{0}>."",""Berichten worden al automatisch verwijderd in <#{0}>.""";;;;;;
+"purger_removed,""No longer automatically removing messages in <#{0}>."",""Berichten worden niet meer automatisch verwijderd in <#{0}>.""";;;;;;
+"purger_no_exists,""There's no automatic message removal in <#{0}>."",""Er worden geen berichten verwijderd in <#{0}>.""";;;;;;
+"purger_list_title,""List of channels with automatic removal of messages and max age:"",""Lijst van kanalen waar berichten automatisch worden verwijderd en de maximale leeftijd:""";;;;;;
+"purger_list_item,""  - <#{0}>: {1} day(s)"",""  - <#{0}>: {1} dag(en)""";;;;;;
+"channel_is_voice,""You should provide a text channel, not a voice channel"",""Geef een tekst kanaal in plaats van een voice kanaal, foemp!""";;;;;;
+"open_source_brief,""Let a user know the bot is open source."",""Laat een gebruiker weten dat de bot open source is.""";;;;;;
+"open_source_usage,""[@user]"",""[@gebruiker]""";;;;;;
+"open_source_help,""Let a user know the bot is open source. \n If there is no user tagged, the bot will send a general message. \n If a user is tagged, the bot will also tag the user in it's message."",""Laat een gebruiker weten dat de bot open source is. \n Als er geen gebruiker is genoemd, zal er een algemene boodschap verschijnen. \n Als er een gebruiker is genoemd, zal de bot deze gebruiker ook taggen.""";;;;;;
+"open_source_message_tag,""Hi {0}, this bot is open source. You can help me become a better me by opening issues or writing software that I need to function properly. If you don't know where to find me, look here: https://github.com/nerdland-unofficial-fans/nerdlandbot."",""Hallo {0}, deze bot is open source. Je kan helpen om me beter te maken door issues aan te maken of software te schrijven die ik nodig heb om te functioneren. Als je niet weet waar je me kan vinden, kan je hier beginnen: https://github.com/nerdland-unofficial-fans/nerdlandbot.""";;;;;;
```

Comment:
I would also add the link to the a new issue in the message with tag, like you did on line 136